iron-fit-behavior.html

```diff
@@ -246,6 +246,22 @@
       return this.horizontalAlign;
     },
 
+    /**
+     * The browser scrollbar size, allowing overflowing containers to compensate.
+     */
+    get _scrollbarSize() {
```

Comment:
Can you do this only once and save it as a variable? Right now this gets run every time `position()` is invoked (which can be lots of times, e.g. at each scroll event)

Maybe just add this into the `properties` object, and make it a read-only, shared property: 

```js
properties: { 
  //...

  /**
   * @private
   */
  _scrollbarSize: {
    type: Number,
    readOnly: true,
    // Shared value across instances.
    value: (function computeScrollbarSize() {
      // your implementation here
    })(),
  }
```
@e111077 are IIFEs ok for modulizer?

Reply:
I think they are fine as long as they don't do a lot of crazy namespace things inside of them.